lib/processLayerStyles.js

```diff
@@ -72,10 +80,61 @@ const layerFill = (layerStyle) => {
 
         }
 
+        if (fill.fillType == FILL_TYPE.IMAGE && fill.isEnabled == true) {
+          // Copy the image from the sketch file to the output folder
+          let imageOutputFolder = outputFolder + "/assets/Assets.xcassets"
+          let imageName = fill.image._ref.split("/").pop()
+          const name = imageName.split(".")[0]
+
+          // Note - this will be async, but we just carry on anyway because we don't use any of the output.
+          fs.mkdirSync(imageOutputFolder, { recursive: true })
+          sketchFile.contentsPath().then(contentsPath => {
+            // Create the asset bundle    
+            prepareAssetBundle(imageName, imageOutputFolder)
+            // Awful, but works around a bug that appears to be in `sketchFile.contentsPath()` where we try to read the 
+            // contents of the folder but it is empty, resulting in a ENOENT.
+            setTimeout(() => {
+              fs.copyFileSync(contentsPath + "/" + fill.image._ref, imageOutputFolder + `/${name}.imageset/` + imageName)
+            }, 1000)
+          })
+
+          // 0: do nothing
+          // 1: fill
+          // 2: stretch
+          // 3: fit
+          switch (fill.patternFillType) {
```

Comment:
seeing as we have the nice FILL_TYPE enum could we have a PATTERN_FILL_TYPE enum too?